src/assets/tool-selector.json

```diff
@@ -0,0 +1,114 @@
+{
```

Comment:
We're going to need to have a dev version of this. Can we do something like `src/assets/tool-selector.dev.json` and `src/assets/tool-selector.prod.json`?

Reply:
I see this has been addressed in an later PR.